app/[...slug]/page.tsx

```diff
@@ -30,7 +30,7 @@ async function getStoryData(params: { slug: string[] }) {
   let slug: string = params.slug ? params.slug.join('/') : 'home';
   let sbParams: ISbStoriesParams = {
     version: activeEnv === 'development' ? 'draft' : 'published',
-    cv: Date.now(),
+    cv: activeEnv === 'development' ? Date.now() : undefined,
```

Comment:
I found that I need the Date.now() when in dev mode so that I can see the changes after hitting "Save" or "Published" on storyblok.